docs/api/browser-window.md

```diff
@@ -190,6 +190,7 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
   * `paintWhenInitiallyHidden` Boolean (optional) - Whether the renderer should be active when `show` is `false` and it has just been created.  In order for `document.visibilityState` to work correctly on first load with `show: false` you should set this to `false`.  Setting this to `false` will cause the `ready-to-show` event to not fire.  Default is `true`.
   * `frame` Boolean (optional) - Specify `false` to create a
     [Frameless Window](frameless-window.md). Default is `true`.
+  * `webContents` WebContents (optional) - Specify `webContents` if there is any existing one.
```

Comment:
I don't think we want to document this as it implies that users are creating their own webContents which isn't documented and is not officially supported.

There is a trade-off here though that `nativeWindowOpen` requires you pass this webContents into the constructor.  There is some work that @loc is doing to make that not-be-a-thing any more though so I'd rather hold for that solution

Reply:
@MarshallOfSound Thanks for the feedback! I am fine with removing this comment since the other part is still a progress for fixing `newGuest` type definition.

Some more context here to explain why we need this.
Our main goal is to get the reference of a new created `BrowserWindow` by calling `window.open`. The example code mentioned in the [doc](https://electronjs.org/docs/api/web-contents#event-new-window) is exactly what we need.

Based on the example code, it looks like there are two missing type definitions:
1. `newGuest` in `new-window` event
2. `webContents` in `BrowserWindow` constructor options.

> I don't think we want to document this as it implies that users are creating their own webContents which isn't documented and is not officially supported.

Does this mean only assigning a customized `webContents` is not officially supported? but `newGuest` usage is fine?

Since it is using in our code already, I would like to know what's the plan for the above example. Would it be deprecated soon or using a different way to achieve the purpose I mentioned above?